src/client-app/src/models/Pieces/bishop.ts

```diff
@@ -11,12 +12,22 @@ export class Bishop extends Piece {
         super(type, color, image);
     }
 
+    validBishop(gameState: Game, {row, col}: Coordinate): Coordinate[]{
```

Comment:
I think it would be better to set these methods as `private` since they will only be used in their respective class.